### PR TITLE
(DOCS-9182) Add workflows widget param info

### PR DESCRIPTION
**Pull request by neko-dd**
### What does this PR do? What is the motivation?
Adds info for the Monitor widget where you add workflows, specifically around the new input parameters flow.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by buraizu** [comment]
Created DOCS-9254 to review

File: content/en/service_management/workflows/trigger.md [hunk 1]
```diff
@@ -69,15 +70,22 @@ To trigger a workflow from a monitor, you must first add a monitor trigger to yo
...
-Add the workflow to your monitor:
+### Add the workflow to your monitor
+
 1. Navigate to the [**Monitors** page][2] in Datadog.
 1. Find the monitor you'd like to use to trigger the workflow and edit it, or create a new monitor.
-1. In the **Configure notifications & automations** section, click **Add Workflow**.
-1. Use the workflow mention name to search for your workflow and select it from the drop-down. Only workflows with monitor triggers appear in the list.
-1. Optionally, to pass trigger variables into the workflow, use a comma-separated list with the syntax `@workflow-name(key=value, key=value)`. For example, `@workflow-my-workflow(name="Bits", alert_threshold=threshold)`
+1. In the **Configure notifications & automations** section, click **+ Add Workflow**.
+1. Use the workflow mention name to search for your workflow and select it from the drop-down. Only workflows with monitor triggers appear in the list.<br>A mention for the monitor appears in the notification message field, in the format `@workflow-name` if it takes no input parameters or `@workflow-name(param="")` if it takes input parameters.
```
Comment (estherk15) [inline content/en/service_management/workflows/trigger.md:78]:
```suggestion
1. Use the workflow mention name to search for your workflow and select it from the dropdown. Only workflows with monitor triggers appear in the list.<br>A mention for the monitor appears in the notification message field, in the format `@workflow-name` if it takes no input parameters or `@workflow-name(param="")` if it takes input parameters.
```

**Review by estherk15 — APPROVED** [review]
Thanks for updating the monitors docs!

**Pull request merged**